src/main/java/de/zalando/zmon/scheduler/ng/config/SchedulerConfig.java

```diff
@@ -69,15 +70,18 @@
 
     String dummyCities = null; // "dummy_data/cities.json"
 
-    // Mapping based on check url prefix
+    // DEPRECATED: Mapping based on check url prefix
     Map<String, String> queueMappingByUrl = new HashMap<>();
 
-    // Map certrain check IDs to queue
+    // DEPRECATED: Map certain check IDs to queue
     Map<String, List<Integer>> queueMapping = new HashMap<>();
 
-    // Map certrain properties to queues e.g. "dc":"gth" => "dclocal:gth"
+    // Map certain entity properties to queues e.g. "dc":"gth" => "dclocal:gth"
     Map<String, List<Map<String,String>>> queuePropertyMapping = new HashMap<>();
 
+    // Maps chosen check/trial run attributes to queues
+    private Map<String, List<Map<String, Object>>> genericQueueMapping = new HashMap<>();
```

Comment:
How does it look in serialized form? No way to simplify?

Reply:
So the JSON-serialized example would be:
```
redis_queue_name_1:
  - some_matching_check_property: check_matching_value
     some_other_property_that_means_bool_and: some_other_matching_value
  - some_other_condition_property: some-val
    ...
redis_queue_name_2: ...
```
One idea could be to flatten one level and have just a list of maps where there is `queue:` among conditions:
```
 - queue: queue_where_it_goes
    some_property: some_value
...
```
but then queue would lie on the same level as conditions and that could be unnecessarily confusing.
Do you have any other ideas how it might look like?